libmscore/synthesizerstate.cpp

```diff
@@ -81,7 +81,7 @@ bool SynthesizerState::isDefaultSynthSoundfont()
       SynthesizerGroup fluid = group("Fluid");
       SynthesizerGroup zerberus = group("Zerberus");
       if (zerberus.size() == 0 && fluid.size() == 1) {
-            if (fluid.front().data == "FluidR3Mono_GM.sf3")
+            if (fluid.front().data == "MS_General.sf3")
```

Comment:
Wouldn’t it make sense to compare both, so that old scores still are considered as using only the default soundfont?

Reply:
Not really. Old scores using the default soundfont will have MS_General.sf3 as their soundfont and not FluidR3Mono_GM.sf3 I believe, except if the user saved the configuration in the score.